requirements.txt

```diff
@@ -0,0 +1,2 @@
+opencv-python
```

Comment:
Do we really need it on the top level? I changed the one that we have inside "model_api/python/"

Reply:
Removed